src/app/core/models/netsuite/netsuite-configuration/netsuite-advanced-settings.model.ts

```
@@ -67,7 +68,19 @@ export type NetsuiteAdvancedSettingAddEmailModel = {
 
 export class NetsuiteAdvancedSettingModel extends HelperUtility {
   static getDefaultMemoOptions(): string[] {
-    return ['employee_email', 'merchant', 'purpose', 'category', 'spent_on', 'report_number'];
+    return AdvancedSettingsModel.getDefaultMemoOptions();
+  }
+
+  static getMemoOptions(exportSettings: NetSuiteExportSettingGet, appName: AppName): string[] {
+    const defaultOptions = this.getDefaultMemoOptions();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)